validation/src/main/scala/hmda/validation/rules/lar/quality/Q005.scala

```diff
@@ -0,0 +1,18 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+object Q005 extends EditCheck[LoanApplicationRegister] {
+  override def apply(lar: LoanApplicationRegister): Result = {
+    when((lar.purchaserType is containedIn(List(1, 2, 3, 4))) and
+      (lar.loan.propertyType is containedIn(List(1, 2)))) {
+      lar.loan.amount is lessThanOrEqual(1203)
+    }
```

Comment:
These values (i.e. 1203) are probably going to change through time (they have in the last few years). We need a mechanism to make that configurable. I suggest adding this to `application.conf` in the validation project with a setting, and providing the ability to override the value with an environment variable. Something like this:

`hmda.validation.quality.Q005.loan.amount=1203`
`hmda.validation.quality.Q005.loan.amount=${?Q005_LOAN_AMOUNT}`

You can then read the value by doing something like:

```scala
val config = ConfigFactory.load()
val loanAmount = config.getInt("hmda.validation.quality.Q005.loan.amount")
```